### ISSUE TITLE
Compiling problem with last couple of changes

### ISSUE THREAD

**Issue by mrdude2478** [Rinnegatamante/vitaGL#17]
Unsure if this is because I'm compiling on windows, however issues arise with vitagl.c

When on line 941 using this code: SCE_GXM_DEPTH_STENCIL_FORMAT_DF32M_S8,

fails to compile - removing the M

IE: SCE_GXM_DEPTH_STENCIL_FORMAT_DF32_S8,

This lets is compile fine on windows, is this an error with windows or the code?

**Comment by Rinnegatamante** [comment]
Update vitasdk.

**Comment by mrdude2478** [comment]
Thanks

**Comment by shadowbrony33** [comment]
reinstall the VitaSDK with the latest version

or

get the latest headers from here: https://github.com/vitasdk/vita-headers

**Comment by mrdude2478** [comment]
I've totally removed the vitasdk and reinstalled with todays version (01/03/18), still getting the same error though. Weird, the builds from about 3 days ago work fine though - everything else I use seems to compile fine. I'll try using ubuntu instead of windows and see if that makes a difference.

**Comment by Rinnegatamante** [comment]
https://github.com/vitasdk/vita-headers/commit/6c353b0be374a0e762af53e5a7f3ddf3bf90eb3b

You're clearly using an outdated vitasdk.

**Comment by mrdude2478** [comment]
Thanks, first I tried updating with: vitasdk-update which seemed to update. Then I removed the vitasdk folder completely and re-installed. Still the same error. I've had to manually download the headers and put them in the vitasdk folder - now everything is working fine.

Thanks, weird how updating the sdk never updated properly though, unless they didn't include the new files???

I'm sorted now though so thanks again.
